Mise à jour de l’adresse postale, fix #2

davidbgk opened the pull request.

davidbgk committed:
Mise à jour de l’adresse postale, fix #2

MattiSG commented on _layouts/default.html:
```diff
@@ -19,9 +19,8 @@ <h1><a href="/">Association Agile France</a></h1>
 		<footer>
 			<p>
```
L'adresse postale ainsi que l'adresse email devraient être englobées dans un [élément `<address>`](https://www.w3.org/TR/2014/PR-html5-20140916/sections.html#the-address-element) au lieu du `<p>`. Cet élément devrait également avoir un attribut [`translate="no"`](https://www.w3.org/TR/2014/PR-html5-20140916/dom.html#the-translate-attribute) pour en empêcher la traduction automatique.

davidbgk replied:
Done with the email but I disagree with the postal address:

> The address element must not be used to represent arbitrary addresses (e.g. postal addresses), unless those addresses are in fact the relevant contact information. (The p element is the appropriate element for marking up postal addresses in general.)

MattiSG replied:
> unless those addresses are in fact the relevant contact information

:wink:

Aren't they? I think both the postal and email addresses are the relevant contact information for the “nearest `body` ancestor”. See the [example on MDN](https://developer.mozilla.org/fr/docs/Web/HTML/Element/address).

I read the spec's emphasis on “no arbitrary address” as in “this is not a generic postal address tag”, not as in “no postal address”.

davidbgk replied:
Fair enough.

MattiSG commented on _layouts/default.html:
```diff
@@ -19,9 +19,8 @@ <h1><a href="/">Association Agile France</a></h1>
 				Association Agile France<br>
-				chez Numa<br>
-				39 rue du Caire<br>
-				75002 Paris<br>
```
Il me semblerait utile d'ajouter `France` au vu de la vocation internationale d'AgileFrance 🙂

davidbgk committed:
Post-review fixes: turn email into address element

> The address element represents the contact information for its nearest article or body element ancestor. — See https://www.w3.org/TR/2014/PR-html5-20140916/sections.html#the-address-element for details

Refs #2

davidbgk committed:
Postal address considered as a contact information

Refs #2

MattiSG reviewed (dismissed):
🎉

davidbgk commented:
No merge?

MattiSG commented:
Was closed accidentally due to target branch deletion. Reopening.
